src/openrct2-ui/windows/Scenery.cpp

```diff
@@ -528,18 +549,23 @@ class SceneryWindow final : public Window
             const auto tabIndex = static_cast<size_t>(widgetIndex - WIDX_SCENERY_TAB_1);
             if (_tabEntries.size() > tabIndex)
             {
+                auto ft = Formatter();
```

Comment:
prefer to keep the formatters in there own little scope. We used to have an issue with reusing formatters making it super confusing so I'd rather not have that again.

Reply:
done